packages/parser/src/repetition.ts

```
@@ -0,0 +1,244 @@
+};
+
+/**
+ * Parses a quantified repetition operator: expr{n}, expr{n,m}, expr{n,}
+ * Supports exact count, range, and minimum repetitions.
+ */
+export const quantifiedOperator = (): Parser<{ min: number; max?: number }> => {
+  // Parse a positive integer
+  const positiveInt = (): Parser<number> => {
+    return map(
+      oneOrMore(charClass(['0', '9'])),
+      (digits) => parseInt(digits.join(''), 10)
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)